From 28bd201ded5f352da82cea78d22d4d3b2f6258c3 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Mon, 22 Oct 2018 23:00:20 +0200
Subject: [PATCH 1/1] efi_selftest: rename setup_ok

The variable name setup_ok might suggest a boolean with true indicating
OK. Let's avoid the misleading name.

%s/setup_ok/setup_status/g

Suggested-by: Simon Glass <sjg@chromium.org>
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_selftest/efi_selftest.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/lib/efi_selftest/efi_selftest.c b/lib/efi_selftest/efi_selftest.c
index fc7866365d..5b01610eca 100644
--- a/lib/efi_selftest/efi_selftest.c
+++ b/lib/efi_selftest/efi_selftest.c
@@ -18,7 +18,7 @@ static const struct efi_boot_services *boottime;
 static const struct efi_runtime_services *runtime;
 static efi_handle_t handle;
 static u16 reset_message[] = L"Selftest completed";
-static int *setup_ok;
+static int *setup_status;
 
 /*
  * Exit the boot services.
@@ -199,8 +199,8 @@ void efi_st_do_tests(const u16 *testname, unsigned int phase,
 		if (test->phase != phase)
 			continue;
 		if (steps & EFI_ST_SETUP)
-			setup_ok[i] = setup(test, failures);
-		if (steps & EFI_ST_EXECUTE && setup_ok[i] == EFI_ST_SUCCESS)
+			setup_status[i] = setup(test, failures);
+		if (steps & EFI_ST_EXECUTE && setup_status[i] == EFI_ST_SUCCESS)
 			execute(test, failures);
 		if (steps & EFI_ST_TEARDOWN)
 			teardown(test, failures);
@@ -278,7 +278,7 @@ efi_status_t EFIAPI efi_selftest(efi_handle_t image_handle,
 	ret = boottime->allocate_pool(EFI_RUNTIME_SERVICES_DATA, sizeof(int) *
 				      ll_entry_count(struct efi_unit_test,
 						     efi_unit_test),
-				      (void **)&setup_ok);
+				      (void **)&setup_status);
 	if (ret != EFI_SUCCESS) {
 		efi_st_error("Allocate pool failed\n");
 		return ret;
-- 
2.19.1

